randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -471,8 +471,8 @@ Extra - asset_id: collision_camera_009
       Morph Ball
   > Pickup (Energy Part)
       All of the following:
-          After Ferenia - Speedboster Slope Maze Blob and Ballspark
-          Flash Shift or Speed Booster Conservation (Beginner) or Use Spin Boost
+          Speed Booster Conservation (Beginner) and Ballspark
```

Comment:
This makes the `Pickup (Energy Part)` not reachable trickless. Should also be the reason why `test_database_collectable` is failing in `test_generator_reach`

Reply:
Any updates? Do we want to throw away the rule of "every pickup needs to be reachable trickless" or do we just change the the db again?